docs/runtime-upgrade.md

```diff
@@ -43,15 +43,15 @@ This must be done before releasing a new version of any of our runtimes to ensur
 
     - Click on **propose**
 
-1. Voting phase. You should make 3 votes in favor with 3 diferent people (modify the CFG amount before to a low number).
+1. Voting phase. You should make 3 votes in favor of 3 different people (modify the CFG amount before to a low number).
```

Comment:
```suggestion
1. Voting phase. The appropriate threshold of votes should be placed in favour of the proposal.
```

Reply:
Following the discussion at https://github.com/centrifuge/centrifuge-chain/pull/1568/files#r1336927624, let's fix this to a more generic sentence that is correct for any of the runtimes and is semantically correct.